src/pages/en/upgrade-astro.mdx

```diff
@@ -0,0 +1,280 @@
+
+```diff
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  mdx: {
+-   extendPlugins: markdown,
+  }
+});
+```
+
+Replace `extendPlugins: defaults` with `extendMarkdownConfig: false` and use the individual options to enable GitHub-Flavored Markdown and SmartyPants if you want to inherit only Astro's default plugins, and not any other Markdown configuration options.
+
+```diff
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  mdx: {
+-   extendPlugins: defaults,
++   extendMarkdownConfig: false,
++   smartypants: true,
++   gfm: true,
+  }
+});
+```
+
+#### Added: More MDX config options to match Markdown
+
+Setting `extendMarkdownConfig` to false now allows you to individually set [every available Markdown configuration option](/en/reference/configuration-reference/#markdown-options) (except `drafts`) separately in your MDX integration config, including `syntaxHighlight`, `shikiConfig` and more.
+
+```astro
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import mdx from '@astrojs/mdx';
+
+export default defineConfig({
+  markdown: {
+    syntaxHighlight: 'prism',
+    remarkPlugins: [remarkPlugin1],
+    gfm: true,
+  },
+  integrations: [
+    mdx({
+      // `syntaxHighlight` inherited from Markdown
+
+      // Markdown `remarkPlugins` ignored,
+      // only `remarkPlugin2` applied.
+      remarkPlugins: [remarkPlugin2],
+      // `gfm` overridden to `false`
+      gfm: false,
+    })
+  ]
+});
+```
+
+### Added: Plugin access to frontmatter
+
+Astro 2.0 gives remark and rehype plugins access to user frontmatter via frontmatter injection. This means `data.astro.frontmatter` is now the _complete_ Markdown or MDX document's frontmatter, rather than an empty object.
+
+This allows plugin authors to modify existing frontmatter, or compute new properties based on other properties. For example, to compute a full image URL based on an `imageSrc` slug in your document frontmatter:
+
+  ```ts
+  export function remarkInjectSocialImagePlugin() {
+    return function (tree, file) {
+      const { frontmatter } = file.data.astro;
+      frontmatter.socialImageSrc = new URL(frontmatter.imageSrc, 'https://my-blog.com/').pathname;
+    };
+  }
+  ```
+
+### Added: `src/content/`
+
+In 2.0, the [Collections API is no longer hidden behind an experimental flag](#content-collections). This API reserves `src/content/` as a special folder. 
+
+If you have an existing `src/content/` folder, it must be renamed to avoid conflicts. This folder can now only be used for [content collections](/en/guides/content-collections/).
+
+## Astro 2.0 Experimental Flags Removed
```

Comment:
Instead of breaking this into `###` subsections, can this section be super minimal instead? Something like:


The following features are no longer marked as experimental, and their experimental flags can be safely removed from your project: 
- `experimental.contentCollections`
- `experimental.prerender`
- `experimental.errorOverlay`

To learn more about these features, check out our dedicated guides on [Content Collections](/en/guides/content-collections/) and Prerender Support.

Reply:
This can be! 

In the rewrite, I added a small bit of text "advertising/celebrating" each in case we wanted to use this (probably highly-traffic'ed) page to draw attention to them. When you look at the latest version, you can decide wether you like this approach, or just want these sections gone!

(I know Chris is team "this is not an update", but I wanted to provide the option, since people might want to explore these new things while they are upgrading anway.)